java/arcs/sdk/wasm/WasmJsInterop.kt

```diff
@@ -249,3 +249,6 @@ fun log(msg: String) {
     write(msg.toWasmString())
     flush()
 }
+
+fun <T, U> combine(handle1: WasmHandleEvents<T>, handle2: WasmHandleEvents<U>):
```

Comment:
How will this extend to more than 2 handles? (Is that still the plan?)

Reply:
Yup! I think from the chat the idea is there will be one for 2 handles, another class for 3, 4, etc. up to some max we set. A bit annoying to implement... but it'll be nicer for developers.